instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/jvm/JvmMetricsCollector.scala

```diff
@@ -259,4 +268,27 @@ object JvmMetricsCollector {
     def sanitize(name: String): String =
       _invalidChars.replaceAllIn(name.toLowerCase, "-")
   }
+  
+  sealed trait ThreadState
+  object ThreadState {
+    case object New extends ThreadState { override def toString: String = "new"}
+    case object Runnable extends ThreadState { override def toString: String = "runnable" }
+    case object Blocked extends ThreadState { override def toString: String = "blocked" }
+    case object Waiting extends ThreadState { override def toString: String = "waiting" }
+    case object TimedWaiting extends ThreadState { override def toString: String = "timed-waiting" }
+    case object Terminated extends ThreadState { override def toString: String = "terminated" }
+    case object Unknown extends ThreadState { override def toString: String = "unknown" }
+    
+    def find(state: String): ThreadState =
+      _threadStateMapping.getOrElse(state, Unknown)
+    
+    private val _threadStateMapping: Map[String, ThreadState] = Map(
+      "NEW"           -> New,
+      "RUNNABLE"      -> Runnable,
+      "BLOCKED"       -> Blocked,
+      "WAITING"       -> Waiting,
+      "TIMED_WAITING" -> TimedWaiting,
+      "TERMINATED"    -> Terminated,
```

Comment:
Please remove this trailing comma, otherwise it will not compile on Scala 2.11

Reply:
done :smile: